.travis.yml

```diff
@@ -121,10 +146,49 @@ jobs:
         - npm run integration-test-run-basic || { npm run integration-test-cleanup; exit 1; }
         - npm run integration-test-run-all || { npm run integration-test-cleanup; exit 1; }
         - npm run integration-test-cleanup
+        - |
+          tagName=`git diff HEAD^ package.json | grep '"version": "' | tail -n 1 | grep -oE "[0-9]+\.[0-9]+\.[0-9]+"` || :
+          if [ -n "$tagName" ]
+          then
+            git tag v$tagName
+            git push https://$GITHUB_TOKEN@github.com/serverless/serverless --tags
+            if [ $TRAVIS_BRANCH = master ]
+            then
+              # Ensure `release-fast-track` branch points latest release
+              git push -f https://$GITHUB_TOKEN@github.com/serverless/serverless master:release-fast-track
+            elif [ $TRAVIS_BRANCH = release-fast-track ]
+            then
+              # Fast forward to `master`
+              git fetch --unshallow
+              git config remote.origin.fetch "+refs/heads/*:refs/remotes/origin/*"
+              git fetch origin master
+              git rebase origin/master
+              git checkout master
+              git merge release-fast-track
+              git push https://$GITHUB_TOKEN@github.com/serverless/serverless
```

Comment:
Who is the author we're using here? Just want to make sure that we have a sounds author appearing in the git logs.

Reply:
It's @serverless-ci altough all eventually rebased commit will be co-authored by original authors

Reply:
Great. Thanks for the clarification :+1:

I just saw that we manually fiddled around with that in our GitHub action so I thought that I call this out here...